client/src/components/CreateForm/NoImageModal/NoImageModal.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import './NoImageModal.scss';
+import Modal from "react-bootstrap/Modal";
+import Button from "react-bootstrap/Button";
+
+/**
+ * Presentational component for the Modal shown when no Images are uploaded
+ */
+export default class NoImageModal extends React.Component {
+
+    render() {
+        const { showing, onClose, onCancel, onContinue } = this.props;
+
+        return (
+            <Modal
+                show={showing}
+                onHide={onClose}
+            >
+                <Modal.Header closeButton>
+                    <Modal.Title>No Images Uploaded</Modal.Title>
+                </Modal.Header>
+                <hr/>
+                <Modal.Body>
+                    <p>At Monuments and Memorials, we're trying our best to provide the most informative data on all of our records.</p>
```

Comment:
This makes it sound like "Monuments an Memorials" is an actual organization or something. I would replace this with simply "We try our best to..."

Reply:
Coming in the next commit.